app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,30 @@ def show
     @post = Post.find(params[:id])
     @comments = @post.comments
   end
+
+  def new
+    post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: post } }
+    end
+  end
+
+  def create
+    post = Post.new(params.require(:post).permit(:title, :text))
+    user = User.find(params[:user_id])
```

Comment:
- [ ] <b>REQUIRED:</b> We don't need this user as we are required to use `current_user` from applicationController. Kindly remove it.